runner/src/server/plugins/engine/page-controllers/ConfirmPageController.ts

```diff
@@ -21,7 +21,23 @@ export class ConfirmPageController extends SummaryPageController {
             const state = await adapterCacheService.getState(request);
             const fund_name = state["metadata"]["fund_name"];
             const round_name = state["metadata"]["round_name"];
-            return redirectTo(request, h, config.eligibilityResultUrl + "/" + fund_name + "/" + round_name);
+
+            let question_answer_short_code;
+
+            // get the question_answer_short_code (round shortname to redirect to) if it was included in the funds eligibility questions
+            for (let [, section] of Object.entries(state)) {
+                if (section && section["redirectToEligibleRound"]) {
```

Comment:
As we use the string `"redirectToEligibleRound"` in multiple places it might be good to extract it out to a fixed constant in case we ever need to update it for some reason.

Something like: 
```js
const REDIRECT_TO_ELIGIBLE_ROUND_QUESTION_KEY = "redirectToEligibleRound"

let eligibleRoundRedirectAnswerShortCode

// ...
```

Reply:
Again, great suggestion, I am on it.